packages/sdk/contractkit/src/wrappers/EpochRewards.ts

```diff
@@ -0,0 +1,63 @@
+import { fromFixed } from '@celo/utils/src/fixidity'
+import { EpochRewards } from '../generated/EpochRewards'
+import { BaseWrapper, proxyCall, valueToBigNumber } from './BaseWrapper'
+
+const parseFixidity = (v: string) => fromFixed(valueToBigNumber(v))
+
+export class EpochRewardsWrapper extends BaseWrapper<EpochRewards> {
+  getRewardsMultiplierParameters = proxyCall(
+    this.contract.methods.getRewardsMultiplierParameters,
+    undefined,
+    (res) => ({
+      max: parseFixidity(res[0]),
+      underspendAdjustment: parseFixidity(res[1]),
+      overspendAdjustment: parseFixidity(res[2]),
+    })
+  )
+
+  getTargetVotingYieldParameters = proxyCall(
+    this.contract.methods.getRewardsMultiplierParameters,
```

Comment:
That should be `getTargetVotingYieldParameters` instead of `getRewardsMultiplierParameters`

Reply:
😱 
great catch